tox.ini

```diff
@@ -80,10 +91,7 @@ html-pretty = behave_html_pretty_formatter:PrettyHTMLFormatter
 
 [flake8]
 exclude = .tox,build,dist,behave_html_pretty_formatter.egg-info
+# see https://github.com/PyCQA/pycodestyle/issues/373
 extend-ignore = E203
 per-file-ignores = ./behave_html_pretty_formatter/__init__.py:F401
-max-line-length = 100
-
-[pylint.main]
-disable = missing-module-docstring,too-many-lines,too-many-instance-attributes,too-many-arguments
```

Comment:
We have changed which warnings we want to disable. Maybe we will split it into multiple files later on.

Reply:
Yeah we can spit it later, no need to rush it for now.